src/types/index.ts

```diff
@@ -87,6 +87,26 @@ export interface RecipientRecievedEventData {
   };
 }
 
+export interface FeeReport {
+  metadata: {
+    chain_id: number; // Network chain ID
+  };
+  distributors: {
+    [distributorAddress: string]: {
+      name: string; // Distributor name
```

Comment:
The distributor doesnt need a name - lets get rid of this property. We can then also get rid of the daily_totals property and make object:
[distributorAddress: string] : Array<data>